ui/packages/catalog-ui-search/src/main/webapp/component/result-item/result-item.collection.tsx

```diff
@@ -24,6 +28,15 @@ const ResultItemCollection = styled.div`
   }
 `
 
+const SolrQueryDisplay = styled.div`
+  > .solr-query {
```

Comment:
✏️ We can make this styled CSS component more granular so that we don't need to specify based on classname. This may also help us remove things like the `!important`